src/models/goal.ts

```diff
@@ -48,6 +58,14 @@ export class Goal extends BaseModel {
   @Column({ type: 'int' })
   progess = 0
 
+  /**
+   * The goals duration, how long the user has to complete their goal.
+   *
+   * Can be on of daily, monthly, weekly, or yearly
+   */
+  @Column({ name: 'goal_duration', type: 'enum', enum: GoalDurations })
+  goalDuration: GoalDurations = GoalDurations.MONTHLY
```

Comment:
Is `MONTHLY` the default here?

Reply:
It is, but that is 99% probably a mistake. I meant to make it `DAILY`

Reply:
I think `DAILY` makes more sense.